tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+
+from exam import fixture
+from freezegun import freeze_time
+
+from sentry.incidents.models import AlertRule
+from sentry.testutils import APITestCase
+
+
+@freeze_time()
+class IncidentCreateEndpointTest(APITestCase):
+    endpoint = 'sentry-api-0-project-alert-rules'
+    method = 'post'
+
+    @fixture
+    def organization(self):
+        return self.create_organization()
+
+    @fixture
+    def project(self):
+        return self.create_project(organization=self.organization)
+
+    @fixture
+    def user(self):
+        return self.create_user()
+
+    def test_simple(self):
```

Comment:
Should you have a test for permission denied? While its handled by the base class it is part of how this endpoint should continue to behave in the future.

Reply:
Will do